internal/syncd/integrations/ecr/listener.go

```diff
@@ -18,6 +23,13 @@ type ImageListener struct {
 	timer   metrics.Histogram
 }
 
+type SQSMessage struct {
```

Comment:
Yes, I thought excluding the struct tags would cause an issue as well. However, I wrote a test case for the parse message function, which unmarshals into the struct and it passes without the json tags.
```
func TestParseMessage(t *testing.T) {
	inputJSON := `
{
	"eventTime": "2019-07-11T14:19:59Z", 
	"repositoryName": "monolith-php", 
	"tag": "78bc9ccf64eb838c6a0e0492ded722274925e2bd",
	"registryId": "723255503624"
}
`

	deployTime, err := time.Parse(time.RFC3339, "2019-07-11T14:19:59Z")
	assert.NoError(t, err)

	expectedMessage := SQSMessage{
		EventTime:      deployTime,
		RepositoryName: "monolith-php",
		Tag:            "78bc9ccf64eb838c6a0e0492ded722274925e2bd",
		RegistryId:     "723255503624",
	}

	inputMessage, err := parseMsg(inputJSON)
	assert.NoError(t, err)
	assert.Exactly(t, expectedMessage, *inputMessage)
}
```

Note that the `SQSMessage` type is renamed to `ecrPushEvent`